packages/native-stack/src/views/FooterComponent.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { ScreenFooter } from 'react-native-screens';
+
+type FooterProps = {
+  children?: React.ReactNode;
+};
+
+export default function FooterComponent({ children }: FooterProps) {
+  return <ScreenFooter collapsable={false}>{children}</ScreenFooter>;
+}
```

Comment:
I don't want this component to be publicly exported from react navigation. I guess it's fine to leave it as it is, right? If someone really wants, he could import it from this file directly but it won't be accessible from the `native-stack` module.